algebra/src/fields/macros.rs

```diff
@@ -190,60 +197,72 @@ macro_rules! impl_Fp {
 
             impl_field_square_in_place!($limbs);
 
+            /// Guajardo Kumar Paar Pelzl
+            /// Efficient Software-Implementation of Finite Fields with Applications to
+            /// Cryptography
```

Comment:
Let's add a link: https://link.springer.com/article/10.1007/s10440-006-9046-1

Reply:
Done but point this out also in thecorresponding related PR